pages/c/_cluster/settings/brand.vue

```diff
@@ -70,12 +81,19 @@ export default {
       uiLogoLight:        '',
       customizeLogo:      false,
 
+      uiColorSetting: {},
+      uiColor:        '',
```

Comment:
Can we default the color to the current primary color?

You can get this with: `getComputedStyle(document.body).getPropertyValue('--primary');`